src/components/styles/Team.css

```
@@ -0,0 +1,97 @@
+@import url('https://fonts.googleapis.com/css2?family=Rufina&display=swap');
+
+.team-boxed {
+    color:rgb(49, 52, 55);
+    background-color:rgb(0, 0, 0);
+  }
+  
+.team-boxed p {
+    color:rgb(0, 0, 0);
+    font-family: 'Rufina', sans-serif;
```

Para el tema de las fuentes recomuendo crear una clase Font con las propiedades mas genericas que se compartan con todas las demas instancias y luego crear variables las cuales carguen despues de la base.
```suggestion
    .font{
        font-family: 'Rufina', sans-serif;
        text-align: center;
        font-size: 24px;
    }
```